domain/src/main/java/uk/gov/hmcts/ccd/definition/store/domain/validation/ValidationErrorMessageCreator.java

```diff
@@ -120,6 +121,8 @@ String createErrorMessage(EventComplexTypeEntityInvalidShowConditionError
     String createErrorMessage(EventComplexTypeEntityDefaultValueError
                                   eventComplexTypeEntityDefaultValueError);
 
+    String createErrorMessage(PublishError publishError);
```

Comment:
we also have a `Publish` column on `CaseEvent`: https://tools.hmcts.net/jira/browse/RDM-10044
Shall we rename these errors to `PublishFieldError` to distinguish them from the error on the publish field on the `CaseEvent` tab?